README.md

```diff
@@ -14,7 +14,7 @@ Read the documentation at: https://symfony.com/bundles/TailwindBundle/current/in
 
 Is this package useful? We're *thrilled* 😍!
 
-A lot of time & effort from the Symfonycasts team & the Symfony community
+A lot of time and effort from the Symfonycasts team & the Symfony community
```

Comment:
Is there any strong reason for changing & -> and here? I mean, & is the correct shortened version of and anyway, moreover later on this line we still have &. Any comments on this change?

Reply:
Ohh, my bad. That shouldn't have been there.

I will remove that in a bit.

Reply:
Sorted. Thank you!